spec/spec_helper.rb

```diff
@@ -90,3 +90,27 @@ def json_payload(file_path)
     JSON.parse(file_fixture(file_path).read)
   end
 end
+
+def stub_octokit_client
+  @octokit_client ||= begin
+    @octokit_client = instance_double("Octokit::Client")
+    allow(Octokit::Client).to receive(:new).and_return(@octokit_client)
+    allow(@octokit_client).to receive(:access_token).and_return(1234)
+    @octokit_client
+  end
+end
+
+def stub_org_request(org_id)
+  org_response = OpenStruct.new(login: "fake org")
```

Comment:
Non-blocking: I'm really interested long term about the speed of OpenStructs and wonder if we might want something more purpose built. But I do like that it has the ability to mix in test-specific params. I don't have a better suggestion or anything, just curious about it.

Reply:
I also stole this approach from education web; I would be interested to know if @renz45 
or @tessgriffin have run into any performance issues with the tests over there.

Reply:
This is pretty interesting https://gist.github.com/jgaskins/3892455

It looks like either of the solutions that have been mentioned here are better than using RSpec Mocks whenever possible.

Reply:
LOL... just noticed that was from 7 years ago... *maybe* it's still true 😝